tests/endpoint/business_test.py

```diff
@@ -13,14 +13,14 @@ def setup_class(cls):
         cls.client = Client(auth)
 
     def test_get_business_builds_correct_params(self):
-        with mock.patch('yelp.client.Client._make_request') as request:
+        with mock.patch('yelp.client.BaseClient.make_request') as request:
```

Comment:
Non-blocking: this mock is used in quite a few places, consider making a decorator or something similar to DRY it out.